### PR TITLE
New contact notifications

### PR DESCRIPTION
**Pull request by kanchan-bluebash**
Added notification icon for showing the new contacts with count. Also, new changes to open if user views the message.
Issue:- https://github.com/publicv-org/publicv/issues/159#issuecomment-908999982

https://user-images.githubusercontent.com/81285549/131665053-8f51dc93-f15d-4843-8646-5c6b54812eb6.mp4

### CODE REVIEW

File: app/controllers/application_controller.rb [hunk 1]
```diff
@@ -2,6 +2,7 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   before_action :set_locale
   before_action :configure_permitted_parameters, if: :devise_controller?
+  before_action :notifications
```
Comment (vishal-bluebash) [inline app/controllers/application_controller.rb:5]: Use current_user condition here

File: app/controllers/application_controller.rb [hunk 2]
```diff
@@ -38,4 +39,9 @@ def tld
   def root_domain_url
     locale.to_s == 'en' ? ENV['EN_SERVER_HOST'] : ENV['IT_SERVER_HOST']
   end
+
+  def notifications
+    @all_contacts = current_user.contacts.where(status: 'new') if current_user.present?
+    @new_contacts = @all_contacts.limit(4) if current_user.present?
```
Comment (vishal-bluebash) [inline app/controllers/application_controller.rb:45]: We don't need limit here, you can pick first records as it is array

**Review by vishal-bluebash — CHANGES_REQUESTED** [review]

File: config/locales/contacts.en.yml [hunk 3]
```diff
@@ -6,3 +6,6 @@ en:
     message: Message
     received_at: Received At
     status: Status
+    details: Contact Details
+    header: Hello, this is
```
Comment (duarme) [inline config/locales/contacts.en.yml:10]: Let's keep things simple, this should simply be "sender". We probably don't even need a label. Every time that something is obvious, we should omit that label.

**Review by duarme — COMMENTED** [review]
Cool feature! Only thing that I'd change is the "Hello this is". It's a nice idea but it's better to keep things as simple as possible. Let's say "Sender: " if needed.

**Pull request merged**